.yamato/scripts/update_from_upstream.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+# Prequisites:
+# This script assumes git and github CLR (gh) are installed.
+
+# Fetch from upstream and rebase our local main on it
+git remote add upstream https://github.com/dotnet/runtime
+git fetch upstream
+git checkout --track origin/main
+git rebase upstream/main
```

Comment:
would it be better to
```
git merge -ff-only upstream/main
```

I want to ensure we fail if a fast forward is not possible.

Reply:
Yes, that sounds like a good idea.